docs/citation.md

```diff
@@ -0,0 +1,18 @@
+# How to Cite
+
+Per [the ScPCA Portal Terms of Use](https://scpca.alexslemonade.org/terms-of-use), users must attribute the data source using any applicable language supplied by data submitters and provided by the Portal when publishing or presenting data or results.
+
+## Citing a project
+
+Please review the project page for the project you are citing for any citation language included in the project abstract or listed publications.
+You can view the project page by substituting the project accession code in the following url: `https://scpca.alexslemonade.org/projects/{project accession}`, e.g., to find the project page for `SCPCP000004`, you can navigate to review https://scpca.alexslemonade.org/projects/SCPCP000004.
+
+## Citing the ScPCA Portal
+
+### APA Format
+
+Alex's Lemonade Stand Foundation Childhood Cancer Data Lab. (n.d.). The Single-cell Pediatric Cancer Atlas Portal. Retrieved (insert access date), from https://scpca.alexslemonade.org/
+
+### MLA Format
+
+The Single-Cell Pediatric Cancer Atlas Portal, Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab, https://scpca.alexslemonade.org/. Accessed (insert access date).
```

Comment:
For consistency, do we want to use handlebars here too?

```suggestion
Alex's Lemonade Stand Foundation Childhood Cancer Data Lab. (n.d.). The Single-cell Pediatric Cancer Atlas Portal. Retrieved {insert access date}, from https://scpca.alexslemonade.org/

### MLA Format

The Single-Cell Pediatric Cancer Atlas Portal, Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab, https://scpca.alexslemonade.org/. Accessed {insert access date}.
```

And, just for amusement:
```
### ICMJE Format
The Single-Cell Pediatric Cancer Atlas Portal [Internet]. Wynnewood (PA): Alex’s Lemonade Stand Foundation Childhood Cancer Data Lab. 2022 - [cited {insert access date}]. Available from: https://scpca.alexslemonade.org/
```
(It's the `[Internet]` followed by a physical location for me.)